setup.py

```diff
@@ -24,6 +26,13 @@
     except ImportError:
         USE_CYTHON = False
 
+    if (here / '.git').exists() and not USE_CYTHON:
```

Comment:
maybe we should also rely on `MULTIDICT_NO_EXTENSIONS`?

Reply:
My strategy is:
- git clone is for development mode. Compilation toolset is required
- Users should install released versions from PyPI.
- Binary wheels are preferred, they don't execute `setup.py` code
- If a binary wheel for the target platform is not available source tarball is used regardless to `MULTIDICT_NO_EXTENSIONS`
- If the toolset is not available pure python installation is executed with a warning to stdout. Better to change stdout -> stderr as a minor improvement

Later I want to consider switching to pure python only if `MULTIDICT_NO_EXTENSIONS` is set to prevent silent performance lose. It enforces something like `MULTIDICT_NO_EXTENSIONS=1 pip install multidict` on machines without compilation toolset when a binary wheel is not available.

Thoughts?

Reply:
Sounds fine, I think. I've pushed minor adjustments. Plz review